homeassistant/components/tts/__init__.py

```diff
@@ -379,6 +378,29 @@ class TextToSpeechEntity(RestoreEntity):
     _attr_should_poll = False
     __last_tts_loaded: str | None = None
 
+    _attr_default_language: str
+    _attr_supported_languages: list[str]
+    _attr_supported_options: list[str] | None = None
+    _attr_default_options: Mapping[str, Any] | None = None
+
+    @classmethod
+    def __init_subclass__(cls) -> None:
```

Comment:
Note that I current suggestion is to remove this method completely and check in `async_internal_added_to_hass` instead.

Reply:
Yes, I still needed to figure out how to test properly.